roles/openshift_master/tasks/main.yml

```diff
@@ -249,7 +249,7 @@
   # Using curl here since the uri module requires python-httplib2 and
   # wait_for port doesn't provide health information.
   command: >
-    curl --silent
+    curl --silent --tlsv1.2
```

Comment:
@sdodson are there other places where not specifying the tls version will be a problem? This might be a good reason to accelerate a change to using the uri module for testing service health as well.